docs/signing.md

```diff
@@ -54,14 +54,17 @@ After formatting the string properly, you can convert it into bytes, for example
 import { char2Bytes } from '@taquito/utils';
 
 const bytes = char2Bytes(formattedInput);
-const payloadBytes = '05' + '0100' + char2Bytes(bytes.length) + bytes;
+const payloadBytes = '0x' + '05' + '01' + `00${char2Bytes((bytes.length / 2).toString())}` + bytes;
```

Comment:
could you also make sure that the examples compile properly? (post a screenshot or something)

Reply:
i added a test that runs the exact same code here 
it doesnt check the string is what it should be but it checks that the hex dec is valid